app/routes/user.accounts/view.tsx

```diff
@@ -1,13 +1,22 @@
+import { hideNotification } from "@mantine/notifications"
 import { Box, Stack, Text } from "@pokt-foundation/pocket-blocks"
+import { useEffect } from "react"
 import AccountsTable from "./components/AccountsTable"
+import { PENDING_INVITES_NOTIFICATION_ID } from "~/components/RootAppShell/RootAppShell"
 import { Account, User } from "~/models/portal/sdk"
 
 type UserAccountsProps = {
   accounts: Account[]
+  pendingAccounts: Account[]
   user: User
 }
 
-export const UserAccounts = ({ accounts, user }: UserAccountsProps) => {
+export const UserAccounts = ({ accounts, pendingAccounts, user }: UserAccountsProps) => {
+  useEffect(() => {
+    // Close the notification if it's still open
```

Comment:
what notification are you closing here?

Reply:
Invited account notification. We close the notification once the user clicks on the link and goes to the accounts page.